Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:5-slim
```

Comment:
just only because it's thinner.
I even try alpine but it doesn't work.
or what do you think?